internal/constants/files.go

```diff
@@ -19,3 +19,5 @@ const DNSWildcardFalseDomainName = "validateNoWildcardDNS"
 
 // Plain http machine config server port
 const InsecureMCSPort = 22624
+
+const MinimalVersionForNmstatectl = "4.13"
```

Comment:
As I said in the previous comment lets put this in the `staticnetworkconfig` package with a helper that uses it.